src/esp/assets/Attributes.h

```diff
@@ -347,6 +375,18 @@ class AbstractPrimitiveAttributes : public esp::core::Configuration {
   void setIsWireframe(bool isWireframe) { setBool("isWireframe", isWireframe); }
 
  protected:
+  /**
+   * @brief Verifies that @ref val is larger than, and a multiple of, divisor
+   * div
+   * @param val the value to check
+   * @param div the divsior (value to verify is greater than and a multiple of)
+   * - will be either 2 or 4 for primitives value checking
+   * @return whether check passes
+   */
+  bool isValMultOfDiv(int val, int div) {
```

Comment:
`isValueMultipleOfDivisor(int val, int div)`

I think it is easier to parse. Up to you.

Reply:
This could also be a free function since it has no relation to the class (or at least a `static` one).